### PR TITLE
Allow basic theme changes to be done from Django Admin

### PR DESCRIPTION
**Pull request by untari**
Work in progress for issue #343. **Do not merge yet**.

This PR is here to show my progress and be able to receive feedback on my work.

## Task list
- [ ] Extract colors, fonts and Font sizes being used and convert them to CSS variables. These values are in CSS files as well as embedded in JS. 
- [ ] Allow modification of these variables from the backend from ZubHub settings. Add a new item called "Theme" in the area below (see issue #343).

### First try for one value
- [x] Get familiar with Makefile commands.
- [x] Get familiar with codebase.
- [x] Identify one color variable you can modify.
- [x] Create a Button for it in then admin panel.
- [ ] Add code, so the Button modifies the color value'.

### Extend for all the CSS values
- [ ] Extract colors, fonts and Font sizes being used and convert them to CSS variables.
- [ ] Add them to the "Theme" Item.

### CODE REVIEW

File: zubhub_backend/zubhub/zubhub/urls.py [hunk 1]
```diff
@@ -66,6 +66,7 @@
             path('api/help/', HelpAPIView.as_view()),
             path('api/privacy/', PrivacyAPIView.as_view()),
             path('api/faqs/', FAQAPIView.as_view()),
+            path('api/faqs/', ThemeAPIView.as_view()),
```
Comment (Ritsyy) [inline zubhub_backend/zubhub/zubhub/urls.py:69]: Let's choose a new URL for this

File: zubhub_backend/zubhub/zubhub/views.py [hunk 2]
```diff
@@ -240,6 +240,21 @@ class FAQAPIView(ListAPIView):
...
+    queryset = Theme.objects.all()
+    serializer_class = ThemeSerializer
+    permission_classes = [AllowAny]
+    throttle_classes = [GetUserRateThrottle, SustainedRateThrottle]
+
+    def get_object(self):
+        obj = self.get_queryset()[:1]
+        if obj:
+            return obj[0]
+        return None
```
Comment (Ritsyy) [inline zubhub_backend/zubhub/zubhub/views.py]: we can replace this block with - 
`self.get_queryset().last()`

File: zubhub_backend/zubhub/zubhub/urls.py (hunk 1 above)
Comment (untari) [reply zubhub_backend/zubhub/zubhub/urls.py:69]: Hi @Ritsyy  I was thinking to name it *api/theme*. What do you think ?

File: zubhub_backend/zubhub/zubhub/views.py (hunk 2 above)
Comment (untari) [reply zubhub_backend/zubhub/zubhub/views.py]: Hi @Ritsyy I just replaced it. Thanks

**Pull request closed without merging**